packages/adp-tooling/package.json

```diff
@@ -46,6 +46,8 @@
         "@sap-ux/odata-service-writer": "workspace:*",
         "@sap-ux/nodejs-utils": "workspace:*",
         "@sap-ux/i18n": "workspace:*",
+        "@sap-ux/environment-check": "workspace:*",
```

Comment:
@cianmSAP Is there better alternative `checkEndpoints`? I won't expect `@sap-ux/environment-check` to be used outside of Environment Check command.

Reply:
There is no exported function that I'm aware of at the minute, this was the approach taken in the abap deploy config inquirer https://github.com/SAP/open-ux-tools/blob/58c2c2ab7a6a1821362ae4e076463728fdef72a8/packages/abap-deploy-config-inquirer/src/utils.ts#L28

I think odata service inquirer handles them separately.

But yes the environment check is not intended for use like this